website/content/en/docs/golang/olm-integration/olm-cli.md

```diff
@@ -369,9 +369,9 @@ requires the typical setup described above.
 [pkg-manifest]:https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/how-to-update-operators.md
 [operator-bundle]:https://github.com/operator-framework/operator-registry/tree/v1.5.3#manifest-format
 [sdk-olm-design]:https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/sdk-integration-with-olm.md
-[cli-olm-install]:https://github.com/operator-framework/operator-sdk/blob/master/doc/cli/operator-sdk_olm_install.md
-[cli-olm-status]:https://github.com/operator-framework/operator-sdk/blob/master/doc/cli/operator-sdk_olm_status.md
-[cli-generate-csv]:https://github.com/operator-framework/operator-sdk/blob/master/doc/cli/operator-sdk_generate_csv.md
+[cli-olm-install]:/docs/cli/operator-sdk_olm_install
```

Comment:
Can we start moving to URL links everywhere instead of using local ones? Then if you want to read docs in the repo without going to the website, all links still work.
```suggestion
[cli-olm-install]:https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/cli/operator-sdk_olm_install.md
```

Reply:
No, its more important that the website links work than the GH rendering, we plan to go live very soon.

Reply:
But we need to add the full URL cannot be a relative path here for the info appears according to the tag version?

Reply:
The website is rendered all at once, so every internal link points to the same doc version. Ideally we will never have to link to an older website doc version.